bin/elmstatic.js

```diff
@@ -399,25 +401,48 @@ function generateEverything(pages, posts, options) {
     const dotGitContent = Fs.pathExistsSync(dotGitPath) ?
         Fs.readFileSync(Path.join(config.outputDir, ".git")).toString() : null
 
-    log(`  Cleaning out the output path (${config.outputDir})`)
-    Fs.emptyDirSync(config.outputDir)
+    if (options.keepOutputAlive) {
+        ; // do nothing
+    } else {
+        log(`  Cleaning out the output path (${config.outputDir})`)
+        Fs.emptyDirSync(config.outputDir)
+    }
+
```

Comment:
Looks like this is going to be problematic if a post/page is deleted or renamed. What was your plan here? 

I'd also prefer to avoid adding configuration options if possible. This seems like the kind of change that should "just work" for everybody.

Reply:
Hm I see two possible solutions here

1. Be pragmatic, since it is a 'development build' served by elmstatic watch and just keep the deleted and 'previous named' files.

2. Or add handles for 'delete' and 'rename' events from chokidar...

Reply:
Deleting everything is a problem, it leads to unconsitency in the postcss process when you use for example 'live-server' or 'browser-sync'

Reply:
Also the new config entry is passed by default to 'generateEverything' in watch mode...